### DISCUSSION TITLE
666 大佬有没有考虑支持一下公网部署

### DISCUSSION THREAD

**Discussion by whmc76** [xingren23/ComfyFlowApp#4] [General]
给大佬打call

**Comment by xingren23** [comment]
感谢支持，让公网用户访问的思路：将应用分享到应用市场，其他用户可以下载使用。
https://github.com/xingren23/ComfyFlowApp/issues/1

**Reply by whmc76** [reply]
这个算力怎么解决呢，如果只是简单部署，用自己的pc好像不太好整到合法ip，但是为了这个去开一套阿里云也有点冤大头

**Reply by xingren23** [reply]
1，开发者，本地开发工作流，将应用上传到应用市场；
2，使用者，从应用市场下载应用，本地使用（本地有GPU算力）

**Reply by whmc76** [reply]
哦哦这样啊 拓宽了思路 666

**Reply by whmc76** [reply]
还有我用信用卡给你买咖啡但是验证过了一直在转圈不太确定有没有到账

**Reply by xingren23** [reply]
看了一下还收到，不知道是不是有延迟，感谢你买的咖啡~
